light_node.cpp

```diff
@@ -344,16 +344,6 @@ void LightNode::setHaEndpoint(const deCONZ::SimpleDescriptor &endpoint)
         isInitialized = item(RStateColorMode) != nullptr;
     }
 
-    if (manufacturerCode() == VENDOR_PHILIPS && endpoint.deviceId() == DEV_ID_HA_ONOFF_LIGHT)
```

Comment:
Note to myself: We need a public database of known devices descriptors, clusters and endpoints.
For example user can opt-in to export this data to an public endpoint which fills the database.